src/main/resources/plugins/zookeeper/TestbotPlugin.py

```diff
@@ -234,6 +234,11 @@ def runner(self, args, display=True):
                 except ProcessorError, ex:
                     LOGGER.error('Failed to process: %s', str(ex))
                     break
+            else:
+                self.results.append(Event(TIMESTAMP_MILLIS(),
+                                          'zookeeper',
+                                          'zookeeper.%d.mode' % (zid), [], MonitorStatus["red"])

```

Comment:
could you please also add details about the error so that you have more insight about the issue (which node is down) plase?

Reply:
Hi Jeremie,
Analyze cause is already managing the details about the error. Thanks.